server/services/database/profile.js

```diff
@@ -3,19 +3,24 @@ const config = require("../../config");
 const pool = new Pool(config);
 
 const createProfile = (profile) => {
-  return pool
-    .query(
-      "INSERT INTO profile (job_title, bio, location, employment_status) values ($1, $2, $3, $4)",
-      [
-        profile.job_title,
-        profile.bio,
-        profile.location,
-        profile.employment_status,
-      ]
-    )
-    .then((result) => result.rows);
+	return pool
+		.query(
+			"INSERT INTO profile (job_title, bio, location, employment_status) values ($1, $2, $3, $4)",
+			[
+				profile.job_title,
+				profile.bio,
+				profile.location,
+				profile.employment_status,
+			],
+		)
+		.then((result) => result.rows);
+};
+
+const getAllProfile = () => {
```

Comment:
The name of this function should be plural, we could return more than one profile.